coordinator-service/src/disk-coordinator.ts

```diff
@@ -2,11 +2,36 @@ import fs from 'fs'
 
 import { Coordinator } from './coordinator'
 import { Ceremony, LockedChunkData } from './ceremony'
+import { SignedContributionData } from './contribution-data'
+import { SignedVerificationData } from './verification-data'
 
 function timestamp(): string {
     return new Date().toISOString()
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function isContributorData(data: any): data is SignedContributionData {
+    return (
+        data != undefined &&
```

Comment:
nit: use the strict inequality operator here `!==`

...i don't think there are cases where it matters when an operand is `undefined`, but I think it's still useful to consistently use the strict operator.

Reply:
I was using it initially, but I moved to the non-strict one to prevent cases where the field is null and so the check still passes. Is there anything better I should do?